lib/omnibus/packagers/rpm.rb

```diff
@@ -437,7 +437,7 @@ def create_rpm_file
           render_template(resource_path("rpmmacros.erb"),
             destination: "#{home}/.rpmmacros",
             variables: {
-              gpg_name: project.maintainer,
+              gpg_name: "Opscode Packages",
```

Comment:
This may have been marked as `project.maintainer` to allow for community users to set up their own build ecosystem where the `gpg_name` would differ. Is there a way we can override only in our build ecosystem?